onebusaway-android/src/main/AndroidManifest.xml

```diff
@@ -40,9 +40,9 @@
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE_SPECIAL_USE" />
     <!-- ACTIVITY_RECOGNITION API 28 and lower -->
-    <uses-permission android:name="com.google.android.gms.permission.ACTIVITY_RECOGNITION" />
+<!--    <uses-permission android:name="com.google.android.gms.permission.ACTIVITY_RECOGNITION" />-->
```

Comment:
can you include a link to the issue where we discussed this so that someone who sees this in the future knows how to contextualize your choice to comment it out?